core/arch/arm/kernel/sub.mk

```diff
@@ -33,7 +33,7 @@ srcs-y += wait_queue.c
 srcs-$(CFG_PM_STUBS) += pm_stubs.c
 
 srcs-$(CFG_GENERIC_BOOT) += generic_boot.c
-srcs-$(CFG_GENERIC_BOOT) += generic_core_bootcfg.c
+srcs-y += generic_core_bootcfg.c
```

Comment:
Should we remove `generic` from the filename, and rename it as `core_bootcfg_mm.c` ?